### PR TITLE
Migrate maintenance jobs to bootstrap

### PR DESCRIPTION
**Pull request by fejta**
Migrate daily and hourly jobs to bootstrap-maintenance.yaml
Migrate job logic to a script.

### CODE REVIEW

**Comment by fejta** [comment]
@rmmh @ixdy ping

File: jenkins/job-configs/bootstrap-maintenance.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+- job-template:
+    name: 'maintenance-{suffix}'  # Runs on each node noce
```
Comment (ixdy) [inline jenkins/job-configs/bootstrap-maintenance.yaml]: 'noce'?

Comment (fejta) [inline jenkins/job-configs/bootstrap-maintenance.yaml]: Done

**Comment by fejta** [comment]
Thank you for creating a SECURITY_CONTACTS file. Please `/close` the issue.
/close

**Comment by BenTheElder** [comment]
Lol @fejta 🤷‍♂️

**Comment by fejta** [comment]
I think/hope that there's an issue #893 somewhere that actually is about security_contacts

**Pull request merged**